CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Updated
```

Comment:
Is this needed in the backport?

Reply:
yes, as we update the changelog for backport releases... although that's non-linear so perhaps we should re-evaluate the formatting going forward.